app/controllers/segment_constraints/document_type_constraint.rb

```diff
@@ -1,6 +1,24 @@
+require "open-uri"
+
 class DocumentTypeConstraint
+  DOCUMENT_TYPE_YAML = "https://raw.githubusercontent.com/alphagov/rummager/master/config/govuk_index/migrated_formats.yaml".freeze
+
+  # Rummager's migrated formats yml doesn't contain all formats, so we need to hardcode these into our prototype
+  HARDCODED_FORMATS = ["residential_property_tribunal_decision", "task_list", "calculator", "field_of_operation", "html_publication", "imported", "ministerial_role", "organisation", "person", "policy_area", "services_and_information", "take_part", "topical_event", "topical_event_about_page", "working_group", "world_location", "worldwide_organisation", "corporate_information_page", "closed_consultation", "consultation_outcome", "esi_fund", "external_content", "national", "news_article", "official", "open_consultation", "statistics_announcement", "topic", "world_location_news_article"]
```

Comment:
Something I've been bitten by in the past is that [govuk-lint](https://github.com/alphagov/govuk-lint/blob/master/configs/rubocop/gds-ruby-styleguide.yml#L308) makes you change arrays of single words to a word array.